src/render/drm_format_set.zig

```diff
@@ -0,0 +1,34 @@
+const std = @import("std");
+const c = @cImport(@cInclude("wlr/render/drm_format_set.h"));
+
+pub const DrmFormat = extern struct {
+    // Use comptime introspection to get the alignment of arch dependent field
+    format: u32 align(std.meta.fieldInfo(c.wlr_drm_format, .format).alignment),
```

Comment:
To avoid the footgun here due to the upstream zig bug, we could add a comptime assert that the alignment is at least 8, which should prevent any ABI mismatches which may be the most painful kind of bug to debug.
```
comptime {
    // This may fail for 32 bit targets due to an upstream `zig translate-c` bug.
    // See https://github.com/ziglang/zig/issues/10944
    std.debug.assert(std.meta.fieldInfo(c.wlr_drm_format, .format).alignment >= @alignOf(u64));
}